examples/web-form-with-express/public/client.js

```diff
@@ -0,0 +1,136 @@
+// This file is run by the browser each time your view template is loaded
+
+/**
+ * Define variables that reference elements included in /views/index.html:
+ */
+
+// Forms
+const dbForm = document.getElementById("databaseForm")
+const pageForm = document.getElementById("pageForm")
+const blocksForm = document.getElementById("blocksForm")
+const commentForm = document.getElementById("commentForm")
+// Table cells where API responses will be appended
+const dbResponse = document.getElementById("dbResponse")
+const pageResponse = document.getElementById("pageResponse")
+const blocksResponse = document.getElementById("blocksResponse")
+const commentResponse = document.getElementById("commentResponse")
+
+/**
+ * Functions to handle appending new content to /views/index.html
+ */
+
+// Appends the API response to the UI
+const appendApiResponse = function (res, el) {
+  console.log(res)
+
+  const result = document.createElement("p")
+  result.innerHTML = "Result: " + res.message
+  el.appendChild(result)
+  // See browser console for more information
+  if (res.message === "error") return
+
+  const id = document.createElement("p")
+  id.innerHTML = "ID: " + res.data.id
+  el.appendChild(id)
+
+  if (res.data.url) {
+    var url = document.createElement("a")
```

Comment:
super nit: can we use `let` instead of `var`?

```suggestion
    let url = document.createElement("a")
```